src/controller/java/templates/ClusterInfo-java.zapt

```diff
@@ -15,11 +15,12 @@ import chip.clusterinfo.DelegatedClusterCallback;
 import chip.clusterinfo.ClusterCommandCallback;
 import chip.clusterinfo.CommandResponseInfo;
 import chip.devicecontroller.ChipClusters.DefaultClusterCallback;
-
+import chip.devicecontroller.ClusterReadMapping;
+import chip.devicecontroller.ClusterWriteMapping;
 
 public class ClusterInfoMapping {
 
-  public class DelegatedCharStringAttributeCallback implements ChipClusters.CharStringAttributeCallback, DelegatedClusterCallback {
+  static public class DelegatedCharStringAttributeCallback implements ChipClusters.CharStringAttributeCallback, DelegatedClusterCallback {
```

Comment:
```suggestion
  public static class DelegatedCharStringAttributeCallback implements ChipClusters.CharStringAttributeCallback, DelegatedClusterCallback {
```
This is the typical order of keywords that Java style should follow, please do the same with the other callbacks